README.md

```diff
@@ -72,6 +72,16 @@ exclude_paths: ['vendor/assets/javascripts']
 ...
 ````
 
+## Rake integration
+To use jshint in your default Rake config, just add it to the list of default tasks. For example, this configuration will run jshint in development or test environments.
+````ruby
+# your-rails-project/Rakefile
+if %w(development test).include? Rails.env
+  task(:default).clear
+  task default: [:spec, :rubocop, :jshint]
```

Comment:
Rather than clearing and redefining the task, it's better to add `jshint` as a dependency of the default task:

```ruby
task default: :jshint
```

The advantage of doing it this way is that (a) it doesn't matter which order you define them in, and (b) definitions in separate files under `lib/tasks` won't stomp on each other.

For example, if you have `lib/tasks/jshint.rake`:

```ruby
if %w[development test].include? Rails.env
  task default: :jshint
end
```

and `lib/tasks/rubocop.rake`:

```ruby
if %w[development test].include? Rails.env
  require 'rubocop/rake_task'
  RuboCop::RakeTask.new

  task default: :rubocop
end
```

then `rake` will run rubocop and jshint in addition to any existing Test::Unit or RSpec tasks.